### PR TITLE
Corrects after review, validation day 28

### PR DESCRIPTION
**Pull request by hrodberaht**

### CODE REVIEW

File: src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js [hunk 1]
```diff
@@ -31,20 +32,25 @@ const asyncValidate = values => schema.validate(values, { abortEarly: false })
     throw errorsReduxForm;
   });
 
+const inputsClassNames = error => classNames({
+  'form-input': true,
```
Comment (ar-mac) [inline src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js]: There is a different, simpler way of defining classes that are added all the time.
I see you used it below

File: src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js [hunk 2]
```diff
@@ -57,7 +63,7 @@ const renderAuction = (fields, error) => (auction, index) => (
       component={renderField}
     />
     <button
-      className="btn danger-btn"
+      className={classNames('btn', 'danger-btn')}
```
Comment (ar-mac) [inline src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js]: There is no need to use `classNames` when you do not have conditionally set class names.

File: src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js [hunk 3]
```diff
@@ -37,14 +38,21 @@ const renderField = ({
   meta: { touched, error },
 }) => (
   <React.Fragment>
-    <input className={`form-input${error ? ' form-error' : ''}`} {...input} type={type} />
+    <input
+      className={classNames({
+        'form-input': true,
```
Comment (pokiujf) [inline src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js]: I meant
`classNames('form-input', {'form-error': error})`

Comment (ar-mac) [inline src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js]: I meant
classNames('form-input', {'form-error': error})
That way form-input is added always, and form-error conditionally

**Review by ar-mac — APPROVED** [review]

**Pull request merged**